### DISCUSSION TITLE
Protect my home network

### DISCUSSION THREAD

**Discussion by khaledez** [exein-io/pulsar#304] [Q&A] [answered]
Hello,
Can pulsar be used to protect my home network from outsiders attacks on the main router? if so, how should I do it?
assuming I have an arm-based custom router

**Comment by banditopazzo** [comment, marked as answer]
Hi, if I understood correctly you want to protect your devices (smartphones, PCs, Smart TVs, etc..) from attacks coming from outside.
If this is correct what are you looking for is a firewall and basically every router has something built-it you can configure, with more or less capabilities depending on how the network appliance is advanced.

pulsar instead is used to protect the device itself from attacks coming from both outside and inside the network or from a local user connected to the machine.
